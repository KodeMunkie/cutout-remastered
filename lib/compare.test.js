/* eslint-env jest */

const baboon = require('baboon-image');
const lena = require('lena');
const compare = require('./compare');

it('correctly compares two identical images', () => {
  const rmse = compare(lena, lena);
  expect(rmse).toBe(0);
});

it('correctly compares two different images', () => {
  const rmse = compare(lena, baboon);
  expect(rmse).toMatchSnapshot();
});
