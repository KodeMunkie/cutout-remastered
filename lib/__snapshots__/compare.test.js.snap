// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly compares two different images 1`] = `0.31274395085329404`;
